store/mongo/main_test.go

```diff
@@ -77,5 +95,84 @@ func GetTestDataStore(t *testing.T) *MongoStore {
 // GetTestDatabase as function above returns the test-local database.
 func GetTestDatabase(ctx context.Context, t *testing.T) *mongo.Database {
 	dbName := legalizeDbName(t.Name())
-	return db.Client().Database(mstore.DbFromContext(ctx, dbName))
+	return client.Database(mstore.DbFromContext(ctx, dbName))
+}
+
+type MongoTestInstance struct {
+	Process  *exec.Cmd
+	HostAddr string
+	DbPath   string
+	ShutDown chan struct{}
+}
+
+func NewMongoTestInstance(path string) *MongoTestInstance {
```

Comment:
this is repeated over and over in many `main_test.go` maybe lets put it one day somewhere in a common place?

Reply:
I'm still trying to figure out a clean interface for unit tests - the current implementation in `go-lib-micro` hides the address the `mongod` instance is listening to. Eventually I think we can modify the existing `dbtest` to something that offers a little more flexibility in the unit tests so we can also cover initialization of `mongo.Client` as well.